nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/HandleHttpRequest.java

```diff
@@ -321,6 +326,24 @@
         return Collections.singleton(REL_SUCCESS);
     }
 
+    @Override
+    protected Collection<ValidationResult> customValidate(final ValidationContext validationContext) {
+        final List<ValidationResult> results = new ArrayList<>();
+
+        final Long port = validationContext.getProperty(PORT).evaluateAttributeExpressions().asLong();
```

Comment:
Why are you doing this? Is the `addValidator` call not validating items from the variable registry?

Reply:
I made a unit test using invalid values (passing from the registry), calling the `.assertValid()` was not marking the processor as invalid

Reply:
The validators, most of the time, won't do anything if expression language is used to configure the property. I guess we could improve the validators based on the EL scope of the property: if the scope is Variable Registry, we should be able to validate the property even if EL is used.